api/modules/os/path.py

```diff
@@ -6,6 +6,17 @@ def abspath(path: str) -> str:
 """
   pass
 
+def relpath(basepath: str, targpath: str) -> str:
+  """Return the path of `targpath` relative to `basepath`. On success, the returned path will
+  always be relative to `basepath`, even if `basepath` and `targpath` share no elements. An
+  error is returned if `targpath` can't be made relative to `basepath`.
```

Comment:
```suggestion
  error is raised if `targpath` can't be made relative to `basepath`.
```

Reply:
Or "it's a tiltfile load error" or something. We aren't very consistent in how we describe this :(.